menu/menu_displaylist.c

```diff
@@ -381,15 +381,15 @@ static int menu_displaylist_parse_core_info(menu_displaylist_info_t *info)
          {
             if (core_info->firmware[i].desc)
             {
-               snprintf(tmp, sizeof(tmp), "	%s: %s",
+               snprintf(tmp, sizeof(tmp), "(!) %s: %s",
```

Comment:
you're adding this (!) here
and here https://github.com/libretro/libretro-super/pull/370/files

Won't this result in (!) (!)?